enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/OnboardingServiceImpl.java

```diff
@@ -84,6 +85,11 @@ public class OnboardingServiceImpl extends CommonOnboardingService {
 
     private final ActivationService activationService;
 
+    /**
+     * Configuration data for client integration
+     */
+    private final ConfigurationDataDto integrationConfigDto;
```

Comment:
Ok, IMHO it is quite unusual but why not. In theory, this makes it impossible to reload properties in runtime, which we do not use anyway. Just a note to realize.